lib/flavors.dart

```diff
@@ -15,16 +10,29 @@ enum Flavor {
 class F {
   static Flavor? appFlavor;
 
-  static Env get env {
+  static String get baseUrl {
```

Comment:
Let's remove // ignore: avoid_classes_with_only_static_members
and the class F{}
and use global functions instead of F class methods.
That means having (no F class at all):

```dart
 String get title {
        switch (appFlavor) {
      case Flavor.QA:
        return 'SecretWall QA';
      case Flavor.PROD:
        return 'SecretWall';
      case Flavor.DEV:
        return 'SecretWall DEV';
      default:
        return 'SecretWall';
    }
  }
```